src/components/Sidebar/Sidebar.jsx

```diff
@@ -35,7 +35,7 @@ export default ({
       <div className="sidebar__inner">
         <div className="sidebar__shields">
           <a href="https://github.com/webpack/webpack/releases">
-            <Shield content="npm/v/webpack" label="webpack" />
+            <Shield content="github/package-json/v/webpack/webpack" label="webpack" />
```

Comment:
```suggestion
            <Shield content="github/package-json/v/webpack/webpack" label="webpack | master" />
```

I would suggest to add something similar ! WDYT ?

Also, keeping this as well 

```js
<Shield content="npm/v/webpack" label="webpack | latest" />
```

Reply:
master has only released version in package json no need to make shield bigger, if its in there, its released

Reply:
its a shame you cannot query the npm shield with betas, there is no such flag. Maybe we can ask for this feature in shields repo

Reply:
> its a shame you cannot query the npm shield with betas, there is no such flag. Maybe we can ask for this feature in shields repo

Didnt know about that. Yes I guess its good to have this enhancement from shield